exonum-java-binding-cryptocurrency-demo/src/test/java/com/exonum/binding/cryptocurrency/ApiControllerTest.java

```diff
@@ -218,6 +230,11 @@ void getWalletUsingInvalidKey(VertxTestContext context) {
         })));
   }
 
+  private BinaryMessage getBinaryMessage() {
+    KeyPair keyPair = CRYPTO_FUNCTION.generateKeyPair();
+    return createSignedMessage(keyPair);
```

Comment:
As the handler doesn't care about the message, any can be passed, therefore, it might be sensible to just inline:
```java
newCryptocurrencyTransactionBuilder(CreateWalletTx.ID)
        .buildRaw();
```

A message with an empty payload will be produced ­— and that’s OK here.